bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewHubHandler.java

```diff
@@ -98,7 +99,7 @@ public class HDPowerViewHubHandler extends BaseBridgeHandler {
     private List<Scene> sceneCache = new CopyOnWriteArrayList<>();
     private List<SceneCollection> sceneCollectionCache = new CopyOnWriteArrayList<>();
     private List<ScheduledEvent> scheduledEventCache = new CopyOnWriteArrayList<>();
-    private Instant firmwareVersionsUpdated = Instant.MIN;
+    private Instant UserDataUpdated = Instant.MIN;
```

Comment:
Field name should be camelCase

Reply:
Whoops! Thanks for noticing. However, PR was already merged, so will fix this and other SAT findings after #13175 has been merged.

Reply:
Actually, besides a few findings in tests, this was the only SAT finding, so renamed this single variable within that PR.